pokemon/app.js

```diff
@@ -93,6 +97,14 @@ nextButton.addEventListener('click', () => {
 
 });
 
+function saveToLocalStorage(key, value) {
```

Comment:
nice work implementing your local storage throughout this project, as well as using comments in your pull request to be more detailed about what action you are performing. Will definitely be a useful action if you need to go back and reference your code.